src/driver.jl

```diff
@@ -0,0 +1,53 @@
+struct Graph500Results{T<:Integer}
+    g::Graph{T}
+    SCALE::Integer
+    NBFS::Integer
```

Comment:
Why are you using capitals here? This indicates to me that you've just copied over the matlab code. Make the code Julian!